plugins/prerequisite_stages/lib/samson_prerequisite_stages/samson_plugin.rb

```diff
@@ -3,30 +3,28 @@
 module SamsonPrerequisiteStages
   class Engine < Rails::Engine
     def self.execute_if_unmet_prereq_stages(stage, reference)
```

Comment:
Maybe change the name of this to something else now? e.g. `check_unmet_prerequisite_stages`?

Reply:
done, also renamed prereq -> prerequisite and moved from `Engine` to base